mne_features/bivariate.py

```diff
@@ -347,7 +354,11 @@ def compute_spect_corr(sfreq, data, db=False, with_eigenvalues=True,
            134445/4803/seizure-detection.pdf
     """
     n_channels = data.shape[0]
-    ps, _ = power_spectrum(sfreq, data, return_db=db)
+    if psd_params is not None:
+        psd_params.update({'psd_method': psd_method})
+        ps, _ = power_spectrum(sfreq, data, **psd_params)
+    else:
+        ps, _ = power_spectrum(sfreq, data, psd_method=psd_method)
```

Comment:
avoid the if and just do:

`ps, _ = power_spectrum(sfreq, data, psd_method=psd_method, **psd_params)`